Sources/DolbyIORTSCore/Builder/StreamSourceBuilder.swift

```diff
@@ -54,7 +52,7 @@ final class StreamSourceBuilder {
     }
 
     func addAudioTrack(_ track: MCAudioTrack, mid: String) {
-        let trackItems = supportedTrackItems.filter({ $0.mediaType == .audio })
+        let trackItems = supportedTrackItems.filter({ $0.mediaType == "audio" })
```

Comment:
better to make match case independent

Reply:
Thanks Ramesh, I reverted the mediaType changes to retain them as enum types and are lowercased now at the time of initialisation.